riva/client/tts.py

```diff
@@ -47,6 +51,10 @@ def synthesize(
             language_code (:obj:`str`): a language to use.
             encoding (:obj:`AudioEncoding`): an output audio encoding, e.g. ``AudioEncoding.LINEAR_PCM``.
             sample_rate_hz (:obj:`int`): number of frames per second in output audio.
+            audio_prompt_file (:obj:`str`): an audio prompt file location for zero shot model.
+            audio_prompt_encoding: (:obj:`AudioEncoding`): encoding of audio prompt file, e.g. ``AudioEncoding.LINEAR_PCM``.
+            audio_prompt_sample_rate: (:obj:`int`): number of frames per second in input audio prompt.
+            audio_prompt_quality: (:obj:`int`): quality of output audio, ranging between 1-40.
```

Comment:
Name `audio_prompt_quality` sounds confusing with the description which says "quality of output audio."

Reply:
Added a better description. Changes the variable name to just `quality` since it refers to the quality of generated audio rather than input `audio_prompt` quality.

Reply:
@virajkarandikar good to resolve?